wlnr/commands.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+const kRelayProtocolVersion uint8 = 1
+
+// The commands used in the protocol
+// The names match the names in the Widelands sources
+// Keep this synchronized with relay_protocol.h
+
+// all
+const kHello uint8 = 1
```

Comment:
use iota instead of manually numbering? Also do not repeat const, but use `const (....)` instead.

https://stackoverflow.com/questions/14426366/what-is-an-idiomatic-way-of-representing-enums-in-go

Reply:
I would prefer hardcoded values here since they have to be the same as in Widelands. It would also work with iota but direct values are more explicit.